EESSI-extend-easybuild.eb

```diff
@@ -106,7 +106,6 @@ elseif (os.getenv("EESSI_SITE_INSTALL") ~= nil) then
   easybuild_installpath = os.getenv("EESSI_SITE_SOFTWARE_PATH")
 else
   -- Deal with user and project installs
-  project_install = os.getenv("EESSI_PROJECT_INSTALL")
```

Comment:
This doesn't look right...

Reply:
Changed in e012bd4d83b0e61cc0445de7a1c3e3507db0d1d1

Reply:
Still here...

Reply:
You're right...! Sorry about that, hopefully fixed in 82da386f0c96a99f9d3a63739e5d1aad13e75362